UPDATING.md

```diff
@@ -130,6 +130,62 @@ session.add(admin)
 session.commit()
 ```
 
+
+## Changes in Google Cloud Platform related operators
+
+Most GCP-related operators have now optional PROJECT_ID parameter. In case you do not specify PROJECT_ID,
+the PROJECT_ID configured in used GCP Connection is used. There will be an exception thrown in case
```

Comment:
Please add the following hyperlink to GCP connection https://airflow.apache.org/howto/manage-connections.html#connection-type-gcp.

Reply:
Good point. Added